.github/workflows/performance.yml

```
@@ -152,7 +162,35 @@ jobs:
         run: npm run test:performance
 
       - name: Print performance tests results
-        run: "node ./tests/performance/results.js"
+        run: node ./tests/performance/results.js
+
+      - name: Check out target commit (target branch or previous commit)
+        run: |
+          if [[ -z "$TARGET_REF" ]]; then
+          git fetch -n origin $TARGET_SHA
```

```suggestion
        run: |
          if [[ -z "$TARGET_REF" ]]; then
            git fetch -n origin $TARGET_SHA
```
Please add indentation for improved visibility.